crates/biome_js_analyze/src/assist/source/use_sorted_keys.rs

```diff
@@ -139,6 +161,43 @@ declare_source_rule! {
     }
 }
 
+/// Determines the nesting depth of a JavaScript expression for grouping purposes.
+fn get_nesting_depth_js(value: &AnyJsExpression) -> u8 {
```

Comment:
A couple of suggestions:
- rename the function, and remove the "js" part, since we are already inside the JS analyser... 😛 
- return [`Ordering`](https://doc.rust-lang.org/std/cmp/enum.Ordering.html) instead of a number, since you're just returning 1 or 0. The code is way safer